CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Feature Requests
+Want a new feature in Mudlet? Let us know about it! We also have a long-running feature requests thread on [our forums](http://forums.mudlet.org/viewtopic.php?f=5&t=92).
+
+# Bug Reports
```

Comment:
Nah, because people will see this file after they've clicked on "New Issue" already.